src/pages/home/ReportScreen.tsx

```diff
@@ -722,6 +722,10 @@ function ReportScreen({route, navigation}: ReportScreenProps) {
             // Early return if the report we're passing isn't in a focused state. We only want to navigate to Concierge if the user leaves the room from another device or gets removed from the room while the report is in a focused state.
             // Prevent auto navigation for report in RHP
             if ((!isFocused && !isReportDetailOpenInRHP) || isInNarrowPaneModal) {
+                if (isInNarrowPaneModal) {
+                    Navigation.goBack(backTo as Route);
+                    return;
+                }
                 return;
             }
```

Comment:
The [simplified logic](https://github.com/Expensify/App/pull/77845#issuecomment-3663524842) from the bot review makes sense.

```suggestion
            if (isInNarrowPaneModal) {
                Navigation.goBack(backTo as Route);
                return;
            }
            if (!isFocused && !isReportDetailOpenInRHP) {
                return;
            }
```

Reply:
I applied it.